spec/lib/instant_verify/verification_request_spec.rb

```diff
@@ -35,6 +36,19 @@
         expect(parsed_body[:Person][:Addresses][0][:StreetAddress2]).to eq('')
       end
     end
+
+    context 'without a uuid_prefix' do
+      let(:attributes) do
+        hash = super()
+        hash.delete(:uuid_prefix)
+        hash
+      end
+
+      it 'does not prepend' do
+        parsed_body = JSON.parse(subject.body, symbolize_names: true)
+        expect(parsed_body[:Settings][:Reference]).to eq(attributes[:uuid])
+      end
```

Comment:
I'm only seeing negative test cases? Can we add a case that checks to make sure we are prefixing correctly when it is present?

Reply:
Nevermind I realized it's in the fixture JSON